src/odbc/src/statement.cpp

```diff
@@ -1029,19 +1109,18 @@ SqlResult::Type Statement::InternalFetchRow() {
     return SqlResult::AI_ERROR;
   }
 
-  if (columnBindOffset) {
-    for (app::ColumnBindingMap::iterator it = columnBindings.begin();
-         it != columnBindings.end(); ++it)
-      it->second.SetByteOffset(*columnBindOffset);
-  }
+  SqlUlen columnBindOffsetValue = columnBindOffset ? *columnBindOffset : 0;
```

Comment:
Should probably leave a comment on why we don't just skip the loop if columnBindOffset is null.

Reply:
Added.